src/utils.rs

```diff
@@ -52,6 +52,20 @@ pub fn random(n: u32) -> u32 {
     })
 }
 
+/// Add additional context to errors
+///
+/// *Note for implementors:* The given closure must only be executed when
+/// `verbose-errors` feature is enabled for this crate!
+pub(crate) trait VerboseErrorExt: Sized {
```

Comment:
I think we could follow `anyhow` / `fehler`'s lead here and call it `Context`, with a method named `context`.

Reply:
Agreed now that it's no longer the name of the feature flag :)